src/Symfony/Component/Messenger/Bridge/AmazonSqs/Tests/Transport/ConnectionTest.php

```diff
@@ -312,4 +315,83 @@ public function testGetQueueUrlNotCalled()
 
         $connection->delete('id');
     }
+
+    public function testLoggerWithoutDebugOption()
```

Comment:
I could replace these tests by just comparing objects of `SqsClient` and just see if they got the same config... that might be better and more "unit"